### DISCUSSION TITLE
:react-native-gesture-handler:compileDebugKotlin FAILED

### DISCUSSION THREAD

**Discussion by Arkan4ik** [software-mansion/react-native-gesture-handler#1761] [Q&A]
I want to use latest `react-native-gesture-handler` but when I want to run it on my android emulator I got an error

`> Task :react-native-gesture-handler:compileDebugKotlin FAILED`

When I downgrade to 1.3.0 it works fine but on version 2.1.0 it does not. 

I added kotlin to the project as it require 
![image](https://user-images.githubusercontent.com/23438777/146075120-cebf4dc1-add8-458f-9bb1-55e8f448fbb0.png)

Could someone help please?

I am using RN 0.63.4

**Comment by 3runoDesign** [comment]
same
